cypress/integration/entitybrowser_spec.js

```diff
@@ -11,62 +11,78 @@ describe('Entity Browser', function() {
 
   describe('ListView', () => {
     it('should load and display essential parts of the list view', function() {
+      cy.contains('Daten werden geladen...', {timeout: 15000})
       cy.get('[title="Vorname"]')
       cy.get('[type="submit"]')
+      cy.contains('Einträgen')
       cy.contains('Aktionen auf Alle')
-      cy.get('[data-list-cell]').should('have.length.above', 1)
+      cy.get('[data-cy=list-cell]').then(el => {
+        cy.wrap(el).should('have.length.above', 10)
+      })
     })
   })
 
   describe('DetailView', () => {
     beforeEach(() => {
-      cy.get('[data-list-cell]').first()
-        .click()
+      cy.get('[data-cy=list-cell]').then(el => {
```

Comment:
isn't it possible to select the first element directly and click it? (`cy.get('[data-cy=list-cell]:first').click()` or similar)

Reply:
Yes. But then it won't wait in an async way.